sktime/transformations/compose.py

```diff
@@ -171,8 +171,13 @@ def __init__(self, steps):
         self._anytagis_then_set("scitype:instancewise", False, True, ests)
         self._anytagis_then_set("fit_is_empty", False, True, ests)
         self._anytagis_then_set("transform-returns-same-time-index", False, True, ests)
-        self._anytagis_then_set("skip-inverse-transform", True, False, ests)
-        self._anytagis_then_set("capability:inverse_transform", False, True, ests)
+        self._anytagis_then_set("skip-inverse-transform", False, True, ests)
+
+        # self can inverse transform if for all est, we either skip or can inv-trasform
```

Comment:
for reviewers: we consider the pipeline to be able to inverse transform if each components can either inverse transform, or are skipped